Sources/IO/IOVideoMixerSettings.swift

```diff
@@ -36,6 +36,8 @@ public struct IOVideoMixerSettings: Codable {
     public let direction: ImageTransform
     /// Specifies the main channel number.
     public var channel: UInt8 = 0
+    /// Specifies if effects are always rendered to a new buffer.
+    public var effectsBuffer: Bool = false
```

Comment:
I see, understood. I accept changes of this magnitude. I'd like the variable name to be changed to "isXXXXEnabled". For example, "isEffectsBufferEnabled" is a bit unclear to me. Do you have any other suggestions for a more appropriate name?

Reply:
We could use "alwaysUseBufferPoolForVideoEffects" or similar. "always" might be a good choice because even when the flag is false, macos will use the buffer pool. And when the flag is true, pixel buffer pool will always be used. Alternatively, we could add a new enum to the settings:
```
enum EffectsRenderMode {
   /// This mode tries to save memory on non-macos platforms by rendering to the video output pixel buffer.
   case optimized
   /// This mode is always using a separate pixel buffer to render the video effects.
   case separateBuffer
}
```

Reply:
Let's go with "alwaysUseBufferPoolForVideoEffects".

Reply:
Updated, please take a look